_posts/2021-03-03-emb-comp.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+title: "Study: Comparing high-dimensional embeddings"
+date: 2021-03-03 12:00:00 -0400
+studyLink: "https://docs.google.com/forms/d/e/1FAIpQLSf6UEl4LaiCMdUvsnnCCM7l9xb2zx6MtWmBdrNg7WzjcknpXA/viewform?usp=sf_link"
+excerpt_separator: <!--more-->
+side_image: "/assets/posts/emb-comp-study.png"
```

Comment:
Run the image through https://tinypng.com/ to reduce the file size.

Reply:
I did already since I remember you asking for that before :)

Reply:
Sweet. Thanks. When I checked the file size, I saw it was 100kb and I wasn't sure.